internal/tiltfile/tiltfile_docker_compose_test.go

```diff
@@ -190,6 +190,58 @@ RUN echo hi`
 	f.assertConfigFiles(expectedConfFiles...)
 }
 
+func TestDockerComposeHonorsDockerIgnore(t *testing.T) {
+	f := newFixture(t)
+	defer f.TearDown()
+
+	df := `FROM alpine
+
+ADD . /app
+COPY ./thing.go /stuff
+RUN echo hi`
```

Comment:
does the fact that this DF has stuff in it have anything to do with the behavior you're testing? nbd but it distracted me a lil 🤷‍♀️

Reply:
nope! removed

Reply:
whoops, it turns out it does:

if you try to ignore a file that's not mounted, the ignore test helper will fail:
```
    tiltfile_test.go:927: Path /private/var/folders/0t/c8t9m67j7qs4y021nl5_3_yr0000gp/T/TestDockerComposeHonorsGitIgnore075810220/foo/tmp is not a dependency of manifest foo
```